README.md

```diff
@@ -38,6 +38,10 @@ Optional Parameters:
                         Paper with code repository password.
   -t GITHUB_ACC_TOKEN, --github_acc_token GITHUB_ACC_TOKEN
```

Comment:
It's not related to this PR but if we can specify this too, I think it'll be great (Both PWC and Github).

Reply:
```suggestion
  -gt GITHUB_ACC_TOKEN, --github_acc_token GITHUB_ACC_TOKEN
```